calyx-py/calyx/py_ast.py

```diff
@@ -371,7 +371,11 @@ def doc(self) -> str:
 
         # Add ref cells if present
         if len(self.ref_cells) > 0:
-            rcs = ", ".join([f"{n}={arg.doc()}" for (n, arg) in self.ref_cells])
+            rcs = ""
+            try:
+                rcs = ", ".join([f"{n}={arg.doc()}" for (n, arg) in self.ref_cells])
+            except AttributeError as e:
+                raise AttributeError(f"Ensure you are passing in a cell to any `ref_` arguments (as opposed to ,say, ports).")
```

Comment:
This is actually unrelated to adding `*=`. But it would have saved me some time debugging things that came up working on this and was already part of the branch, so here it is. Apologies for the sloppy git hygiene, I can remove this and make a separate PR if desired!